
class LoginError(Exception):
    """Error during login process"""

